### PR TITLE
Add users as assets (WIP)

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: src/interfaces.ts [hunk 1]
```diff
@@ -1,8 +1,9 @@
 export interface User {
-  id: number;
+  id?: number;
+  email?: string;
   first_name?: string;
   last_name?: string;
-  email?: string;
+  role?: string;
```
[the remark this answers is not in the document]
Comment (ericnewcomer) [reply src/interfaces.ts:6]: Is there any reason all of these attributes are optional?

[the remark this answers is not in the document]
Comment (rowanseymour) [reply src/interfaces.ts:6]: I vaguely recall there's another part of the code based using this interface with objects that only have `id`

[the remark this answers is not in the document]
Comment (rowanseymour) [reply src/interfaces.ts:6]: now it needs to match https://app.rapidpro.io/api/v2/users

[the remark this answers is not in the document]
Comment (ericnewcomer) [reply src/interfaces.ts:6]: Shouldn't they all at least have an id though?

[the remark this answers is not in the document]
Comment (rowanseymour) [reply src/interfaces.ts:6]: Currently the API treats `email` as the identifier for contacts.. things like the ticket bulk actions endpoint take email.. so in theory you never need `id`

**Pull request closed without merging**